css/localgov-alert-banner.css

```diff
@@ -8,13 +8,21 @@
   background-color: #00856A;
 }
 
+dialog.localgov-alert-banner {
+  margin: auto;
+}
+
+dialog.localgov-alert-banner::backdrop {
+  background-color: rgba(0, 0, 0, 0.8);
+}
```

Comment:
Hmm, I think this should go in full screen alert banner module, not the main stylesheet.

/modules/localgov_alert_banner_full_page/css/full-page-alert-banner-layout.css or full-page-alert-banner-theme.css.